File: beans/ios/Runner/Native/EventChannelRegister.swift
import Flutter

final class EventChannelRegister {

    private let streams:
    [NativeStreamHandler]

    init(
    streams: [NativeStreamHandler]
    ) {

        self.streams = streams
    }

    func register(
    messenger: FlutterBinaryMessenger
    ) {

        streams.forEach {

            $0.register(
                messenger: messenger
            )

        }
    }
}

File: beans/android/app/src/main/kotlin/com/example/beans/EventChannelRegister.kt
package com.example.beans

import io.flutter.plugin.common.BinaryMessenger
class EventChannelRegister(
    private val streams:
    List<NativeStreamHandler>
) {

    fun register(
        messenger: BinaryMessenger
    ) {

        streams.forEach {

            it.register(
                messenger
            )

        }
    }
}